HackIllinois/UI/TableView/Cells/HIBubbleCell/HIEventCell.swift

```diff
@@ -124,20 +124,18 @@ extension HIEventCell {
         pointsLabel.constrain(to: pointsView, topInset: 0, trailingInset: 0, bottomInset: 0, leadingInset: 0)
         pointsLabel.text = "\(rhs.points) Points!"
 
-        let eventType = HILabel(style: .eventType)
-        eventType.text = rhs.eventType
-        if rhs.eventType == "QNA" {
-            eventType.text = "COMPANY Q&A"
-        }
-        eventType.textHIColor = getTagColor(for: rhs.eventType)
-        eventType.refreshForThemeChange()
+        let eventTypeLabel = HILabel(style: .eventType)
+        let eventType = HIEventType(type: rhs.eventType)
+        eventTypeLabel.text = String(describing: eventType)
```

Comment:
using `.description` or "\(eventType)" is preferable

Reply:
Fixed to .description